consumer/config.go

```diff
@@ -4,13 +4,12 @@ import (
 	"time"
 
 	"github.com/Shopify/sarama"
-	cluster "github.com/bsm/sarama-cluster"
 	"github.com/heetch/felice/codec"
 )
 
 // Config is used to configure the Consumer.
 type Config struct {
-	cluster.Config
+	*sarama.Config
```

Comment:
Technically this is a backwardly incompatible change. Do we really need to expose this implementation detail to clients of this package?

Reply:
Yes you are right but the idea behing exposing it is to allow users to customize the configuration following their needs. Wdyt?

Reply:
The fact that we've changed it now without breaking clients (presumably) is a bit of an indicator to me that it's really more of an implementation detail than something we'd like to expose. I'd tend to err on the side of caution - we can always add configuration hooks later, I think, if called for.